cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/monitor/RuntimeMonitor.java

```diff
@@ -93,19 +91,9 @@ private void initializeTopics() throws Exception {
     Set<String> topicsToMonitor = new HashSet<>();
     topicsToMonitor.addAll(Arrays.asList(cConf.get(Constants.RuntimeMonitor.TOPICS).split(",")));
```

Comment:
These are not `topics` right? Better rename the variable, the constant and the config name.